### PR TITLE
[RFC][UN-9605] Add disabled support to uni-input-price

### PR DESCRIPTION
**Pull request by henrmota**
https://uniplaces.atlassian.net/browse/UN-9605

### CODE REVIEW

File: addon/styles/partials/_uni-input-price.scss [hunk 1]
```diff
@@ -9,6 +9,15 @@
     border-color: color('input', 'border-focus');
   }
 
+  &--disabled {
+    color: color('input', 'disabled');
```
Comment (mmelo) [inline addon/styles/partials/_uni-input-price.scss]: use double quotes here please

File: addon/styles/partials/_uni-input-price.scss [hunk 2]
```diff
@@ -9,6 +9,15 @@
...
+    background-color: #f0f1f3;
```
Comment (mmelo) [inline addon/styles/partials/_uni-input-price.scss]: double check if this exists in uniplaces-uniq color palette

File: addon/styles/partials/_uni-input-price.scss [hunk 3]
```diff
@@ -9,6 +9,15 @@
...
+
+    input {
+      background-color: #f0f1f3;
```
Comment (mmelo) [inline addon/styles/partials/_uni-input-price.scss]: same here

**Review by mmelo — CHANGES_REQUESTED** [review]

**Review by mmelo — APPROVED** [review]

File: addon/components/uni-input-price.js [hunk 4]
```diff
@@ -7,6 +7,7 @@ const INPUT_TYPE_NUMBER = 'number';
 
 export default Component.extend({
   classNames: ['uni-input-price'],
+  classNameBindings: ['isDisabled:uni-input-price--disabled'],
```
Comment (AndreJoaquim) [inline addon/components/uni-input-price.js:10]: Add an example of this case to the demo.

**Review by AndreJoaquim — CHANGES_REQUESTED** [review]

**Review by AndreJoaquim — APPROVED** [review]

**Pull request merged**